app/models/manageiq/providers/ansible_runner_workflow.rb

```diff
@@ -181,6 +181,14 @@ def checkout_git_repository
     save!
     _log.info("Checking out git repository to #{options[:git_checkout_tempdir].inspect}...")
     css.checkout_git_repository(options[:git_checkout_tempdir])
+  rescue MiqException::MiqUnreachableError => err
+    miq_task.job.timeout!
+    r_id = options[:extra_vars]["manageiq"]["request"].split('/').last.to_i
+    if MiqRequest.find_by(:id => r_id)
```

Comment:
Probably makes sense to **not** do this lookup twice (you do it again on the next line).

Consider doing the following instead:

```ruby
if miq_request = MiqRequest.find_by(:id => r_id)
  task = miq_request.miq_request_tasks.first
```